src/main/kotlin/racingcar/view/OutputView.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar.view
+
+import racingcar.domain.Car
+import racingcar.domain.Cars
+
+fun showNameInputMessage() {
+    println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).")
+}
+
+fun showCountInputMessage() {
+    println("시도할 회수는 몇회인가요?")
+}
+
+fun showCarsPosition(cars: Cars) {
+    for (car in cars.values) {
+        println("${car.name} : " + "-".repeat(car.position))
+    }
```

Comment:
이런식으로 변경하면 `+` 도 없애고 `println()` 도 없앨 수 있을 것 같아요!

```suggestion
    println(
        cars.values
            .joinToString(separator = "\n", postfix = "\n") { "${it.name} : ${"-".repeat(it.position)}" }
    )
```

Reply:
오 이런 방식도 있었군요! 감사합니다